### PR TITLE
Small doc tweaks.

### PR DESCRIPTION
**Pull request by codeviking**
* Put the example in it's own file.
* Replace `webpage` with `web page`.
* Add spans to the simple example.

@aaronsarnat Please review.

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Examples
+
+## In a web page:
+
+Here's an example, showing how the dependency parse for the sentence `"Sam likes bananas"` can be
+rendered using `hierplane`:
```
Comment (aaronsarnat) [inline EXAMPLES.md:6]: Maybe call out that this is using the CDN link method? Makes sense to me, but I could see it being a bit confusing if the user is also looking at the built-in JSON examples that are formatted/organized slightly differently.

**Review by aaronsarnat — APPROVED** [review]
Just one comment then LGTM

File: EXAMPLES.md (hunk 1 above)
Comment (codeviking) [reply EXAMPLES.md:6]: I think that's implied by the `In a web page` title, right?  Does that make sense?

File: EXAMPLES.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+<!DOCTYPE html><html>
+  <head>
+    <title>Hierplane!</title>
+    <link rel="stylesheet" type="text/css" href="//unpkg.com/hierplane/dist/static/hierplane.min.css">
+  </head>
+  <body>
+    <script src="//unpkg.com/hierplane/dist/static/hierplane.min.js"></script>
```
Comment (codeviking) [inline EXAMPLES.md:15]: Also, technically the CDN reference is here.  I think we're good to go :).

File: EXAMPLES.md (hunk 1 above)
Comment (aaronsarnat) [reply EXAMPLES.md:6]: Works for me. I trust your judgement. :)

**Pull request merged**